### PR TITLE
LRQA-72202 | master

### PR DESCRIPTION
**Pull request by melodywu**
https://issues.liferay.com/browse/LRQA-72202

### CODE REVIEW

**Comment by melodywu** [comment]
ci:test:sf

**Comment by patriciajeanperez** [comment]
@melodywu Manage Organizations permission is found under Account Entry. For our test, we can just add it to the Account Administrator role since the test is using the Account Admin. In the UI, it can be found in `Account Roles > Account Administrator > Define Permissions > Accounts > Account Entry > Manage Organizations`. With JSON, we can use the macro `Permissions.definePermissionViaJSONAPI`. Thanks! Please also see SF errors.

**Comment by melodywu** [comment]
ci:test:sf

**Comment by melodywu** [comment]
> @melodywu Manage Organizations permission is found under Account Entry. For our test, we can just add it to the Account Administrator role since the test is using the Account Admin. In the UI, it can be found in `Account Roles > Account Administrator > Define Permissions > Accounts > Account Entry > Manage Organizations`. With JSON, we can use the macro `Permissions.definePermissionViaJSONAPI`. Thanks! Please also see SF errors.

Done thanks！

File: modules/apps/account/account-test/src/testFunctional/tests/2BAccount.testcase [hunk 1]
```diff
@@ -531,6 +531,11 @@ definition {
 			userLoginEmailAddress = "test@liferay.com",
 			userLoginFullName = "Test Test");
 
+		Permissions.definePermissionViaJSONAPI(
+			resourceAction = "MANAGE_ORGANIZATIONS",
+			resourceName = "com.liferay.account.model.AccountEntry",
+			roleTitle = "Account Administrator");
+
```
Comment (patriciajeanperez) [inline modules/apps/account/account-test/src/testFunctional/tests/2BAccount.testcase:538]: @melodywu roleType wasn't set to Account so it didn't add the permission to the Account Administrator under Account Role. I'll go ahead and update.

**Comment by patriciajeanperez** [comment]
@melodywu I set the roleType to Account to add the permission in the account role. See https://github.com/patriciajeanperez/liferay-portal/pull/534

**Pull request closed without merging**